test/e2e/resources/full-chart-install.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       clientConfig:
         caBundle: Cg==
         service:
-          name: webhook-service
+          name: capi-operator-webhook-service
```

Comment:
where is this service being defined?

Reply:
It is here https://github.com/kubernetes-sigs/cluster-api-operator/blob/main/test/e2e/resources/full-chart-install.yaml#L6421

Reply:
/lgtm